packages/framework/presence/src/presenceDatastoreManager.ts

```diff
@@ -256,9 +256,18 @@ export class PresenceDatastoreManagerImpl implements PresenceDatastoreManager {
 			// If it isn't then, not really a problem; just won't be in provider or quorum list.
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 			const clientId = this.runtime.clientId!;
+			const requestor = message.clientId;
```

Comment:
Want to call it `requestorClientId` to make the semantics of it more clear?  I'm assuming it is some other client's clientId, same as this client has a clientId which we log all over.

Reply:
I did not because clientId is a bit confusing in this context. We do say clientId all over the place and mean client's curernt connection id. We explicitly differentiate within presence space because we mostly work with client session id (also a guid). I could name it requestorClientConnectionId, but that will be confusing and very verbose.
We don't actually care telemetry-wise that it is a client connection id - at least not for any of the metrics I am thinking about (also needs to be written out). What matters is that it is a unique id in response to a particular join (same id used by all respondents if there are multiple respondents).
So genericly named that happens to be client connection id is good for now. (Probably I should add some comments.)